config/packages/test/sylius_uploader.yaml

```diff
@@ -4,8 +4,5 @@ services:
         arguments:
             - "%sylius.uploader.filesystem%"
         factory: ['@knp_gaufrette.filesystem_map', 'get']
-    sylius.image_uploader:
-        class: Sylius\Behat\Service\Uploader\ImageUploader
-        arguments:
-            - "@sylius_test.filesystem"
```

Comment:
Are we sure we don't need to use `sylius_test.filesystem` anymore?

Reply:
Good point 😄